scripts/confirm-multiple.ps1

```diff
@@ -0,0 +1,26 @@
+$title    = 'File Operation'
```

Comment:
I wouldn't include these in the main repo, in some personal repo should be fine and you could add that to your PATH to make it convenient to use